docs/developer_guide/schemas.rst

```diff
@@ -0,0 +1,374 @@
+          "default": true
+        },
+        "spikeextractors_backend": {
+          "type": "boolean",
+          "default": false
+        }
+      },
+      "type": "object",
+      "additionalProperties": false
+    }
+
+A ``ExampleNWBConverter`` that combines these ``DataInterface``s will combine the JSON Schema using the keys of the
+``data_interface_classes`` dictionary:
+
+.. code-block:: python
+
+    from neuroconv import NWBConverter,
+    from neuroconv.datainterfaces import (
+        SpikeGLXRecordingInterface,
+        PhySortingInterface
+    )
+
+    class ExampleNWBConverter(NWBConverter):
+        data_interface_classes = dict(
+            SpikeGLXRecording=SpikeGLXRecordingInterface,
+            PhySorting=PhySortingInterface
+        )
+
+    ExampleNWBConverter.get_source_schema()
+
+
+.. code-block:: json
+
+    {
+      "$schema":"http://json-schema.org/draft-07/schema#",
+      "$id":"source.schema.json",
+      "title":"Source data schema",
+      "description":"Schema for the source data, files and directories",
+      "version":"0.1.0",
+      "type":"object",
+      "required":[],
+      "properties":{
+        "SpikeGLXRecording":{
+          "required":[
+            "file_path"
+          ],
+          "properties":{
+            "file_path":{
+              "format":"file",
+              "type":"string",
+              "description":"Path to SpikeGLX file."
+            },
+            "stub_test":{
+              "type":"boolean",
+              "default":false
+            },
+            "spikeextractors_backend":{
+              "type":"boolean",
+              "default":false
+            },
+            "verbose":{
+              "type":"boolean",
+              "default":true
+            }
+          },
+          "type":"object",
+          "additionalProperties":false
+        },
+        "PhySorting":{
+          "required":[
+            "folder_path"
+          ],
+          "properties":{
+            "folder_path":{
+              "format":"directory",
+              "type":"string"
+            },
+            "exclude_cluster_groups":{
+              "type":"array"
+            },
+            "verbose":{
+              "type":"boolean",
+              "default":true
+            },
+            "spikeextractors_backend":{
+              "type":"boolean",
+              "default":false
+            }
+          },
+          "type":"object",
+          "additionalProperties":false
+        }
+      },
+      "additionalProperties":false
+    }
+
+Conversion schemas options work similarly to source schemas.
+
+.. _metadata_schema:
+
+Metadata Schema
+---------------
+
+Similar to input data, each ``DataInterface`` produces its own metadata schema reflecting
+the specificities of the dataset it interfaces with. The ``DataInterface``-specific metadata schema can be obtained
+via method ``.get_metadata_schema()``. Unlike ``.get_source_schema``, the ``DataInterface`` needs to be initialized
```

Comment:
```suggestion
via method ``.get_metadata_schema()``. Unlike ``.get_source_schema()``, the ``DataInterface`` needs to be initialized
```

For consistency with the previous line

Reply:
![image](https://user-images.githubusercontent.com/51133164/208479418-ef5f95eb-617e-47d5-8048-610c5fa6e0b1.png)